lib/elaborate/elaborate-aliases.ts

```diff
@@ -5,7 +5,7 @@ export function elaborateAliases(file: OFile) {
   for (const alias of file.objectList) {
     if (alias instanceof OAlias) {
       if (alias.name.length === 0) {
-        throw new ParserError(`Alias without name`, alias.range);
+        throw new ParserError(`Alias without name`, alias.range); // TODO: why is this a parser error?
```

Comment:
Make this sound like a TODO which make sense in a issue. ie Change ParserError to addMessage or sth.